examples/dispatch_iface.yml

```diff
@@ -0,0 +1,21 @@
+---
+dispatch:
+    interface-types:
+        - name: vxcan
+          activation: "ip link add $name type vxcan peer $peer"
+          deactivation: "ip link del $name"
+          allowed-variable-names:
+              - peer
+          kernel-index-getter: "cat /sys/class/net/$name/ifindex"
```

Comment:
As I understand this correctly a user can apply just the dispatchs types at a yaml and then use them at different yamls.

If this is the case maybe we should allow to delete dispatchs ? adding something like "state: present" or "state: absent"  under interface-types

```yaml
dispatch:
    interface-types:
        - name: vxcan
           state: absent
```

Reply:
We support so. Just not mention in commit message.